perun-base/src/main/java/cz/metacentrum/perun/core/api/User.java

```diff
@@ -40,7 +37,7 @@ public User(int id, String firstName, String lastName, String middleName, String
 	}
 
 	public User(int id, String firstName, String lastName, String middleName, String titleBefore, String titleAfter,
-			String createdAt, String createdBy, String modifiedAt, String modifiedBy, Integer createdByUid, Integer modifiedByUid) {
+	            String createdAt, String createdBy, String modifiedAt, String modifiedBy, Integer createdByUid, Integer modifiedByUid) {
```

Comment:
No, this indentation is correct, the first level is indented by tab, the second level matches the start of first argument, so it must use spaces. It was indented automatically anyway  :-)